src/Controller/AccountService.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Controller;
+
+class AccountService extends AbstractController
+{
+    public function logout()
+    {
+        session_destroy();
+        header('Location: /');
+    }
```

Comment:
Because of the logout function, it's not only a Service class, it's a real Controller ! So you should rename this class "AccountController"